schemas/business/index.ts

```diff
@@ -0,0 +1,29 @@
+import mongoose from 'mongoose';
+const { Schema } = mongoose;
+
+const businessSchema = new Schema({
```

Comment:
Where is `id` of business ? and id of enab business should be "enab" not random. same to space should be specified as string for enab spaces only.